src/components/common/AlertDropdown.tsx

```diff
@@ -27,14 +28,14 @@ function AlertDropdownComponent(props: Props & ThemeProps) {
     >
       <View style={styles.container}>
         <EntypoIcon name="warning" size={theme.rem(1.25)} style={styles.icon} />
-        <Text style={styles.text}>
-          <Text style={styles.boldText}>
-            {warning
+        <UnscaledText style={styles.text}>
+          <UnscaledText style={styles.boldText}>
+            {(warning
               ? lstrings.alert_dropdown_warning
-              : lstrings.alert_dropdown_alert}{' '}
-          </Text>
+              : lstrings.alert_dropdown_alert) + ' '}
```

Comment:
In the eslint.config.mjs, we have:

```ts
'react-native/no-raw-text': [
        'error',
        {
          skip: ['B', 'EdgeText', 'Paragraph', 'SmallText', 'WarningText']
        }
      ]
```

We should probably add `UnscaledText` to the list.